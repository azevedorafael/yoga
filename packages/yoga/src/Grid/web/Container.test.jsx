import React from 'react';
import { render } from '@testing-library/react';
import { ThemeProvider } from '../..';
import { Container } from '.';

describe('<Container />', () => {
  describe('Snapshots', () => {
    it('should match snapshot with default Container', () => {
      const { container } = render(
        <ThemeProvider>
          <Container />
        </ThemeProvider>,
      );
      expect(container).toMatchSnapshot();
    });
  });
});
