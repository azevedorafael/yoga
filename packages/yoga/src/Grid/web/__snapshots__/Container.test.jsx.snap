// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Snapshots should match snapshot with default Container 1`] = `
.c0 {
  box-sizing: border-box;
  max-width: 1600px;
}

@media (min-width:0px) {
  .c0 {
    margin-left: 20px;
    margin-right: 20px;
  }
}

@media (min-width:1024px) {
  .c0 {
    margin-left: 71px;
    margin-right: 71px;
  }
}

@media (min-width:1742px) {
  .c0 {
    margin-left: auto;
    margin-right: auto;
  }
}

<div>
  <div
    class="c0"
  />
</div>
`;
